src/Symfony/Component/ErrorHandler/Tests/Fixtures/InterfaceWithAnnotatedParameters.php

```diff
@@ -13,14 +13,14 @@ interface InterfaceWithAnnotatedParameters
     public function whereAmI();
 
     /**
-     * @param       $noType
+     * @param       $noType with $dollar after
      * @param callable(\Throwable|null $reason, mixed $value) $callback and a comment
      * about this great param
      * @param string                                          $param (comment with $dollar)
      * @param $defined
      * @param  callable  ($a,  $b)  $anotherOne
      * @param callable (mixed $a, $b) $definedCallable
-     * @param Type$WithDollarIsStillAType $ccc
+     * @param Type$ccc $ddd
```

Comment:
Let's remove this test as we don't want to explicitly support this case, which is an invalid phpdoc anyway.

Reply:
removed!